demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -0,0 +1,276 @@
+    return (
+        <DataGridToolbar>
+            <ToolbarItem>
+                <GridToolbarQuickFilter />
+            </ToolbarItem>
+            <ToolbarItem>
+                <GridFilterButton />
+            </ToolbarItem>
+            <ToolbarFillSpace />
+            {toolbarAction && <ToolbarActions>{toolbarAction}</ToolbarActions>}
+        </DataGridToolbar>
+    );
+}
+
+type Props = {
+    toolbarAction?: React.ReactNode;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    rowAction?: (params: GridRenderCellParams<any, GQLCombinationFieldsTestProductsGridFutureFragment, any>) => React.ReactNode;
+};
+
+export function ProductsGrid({ toolbarAction, rowAction }: Props): React.ReactElement {
+    const client = useApolloClient();
+    const intl = useIntl();
+    const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("ProductsGrid") };
+
+    const columns: GridColDef<GQLCombinationFieldsTestProductsGridFutureFragment>[] = [
+        {
+            field: "titleAndCategory",
+            headerName: intl.formatMessage({ id: "product.titleAndCategory", defaultMessage: "Title and Category" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                return (
+                    <GridCellContent
+                        primaryText={row.title}
+                        secondaryText={
+                            row.category?.title ?? (
+                                <FormattedMessage id="product.titleAndCategory.secondaryText.empty" defaultMessage="No category set" />
+                            )
+                        }
+                    />
+                );
+            },
+            flex: 1,
+            minWidth: 150,
+        },
+        {
+            field: "staticTextExample",
+            headerName: intl.formatMessage({ id: "product.staticTextExample", defaultMessage: "Static text" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                return (
+                    <GridCellContent
+                        primaryText={<FormattedMessage id="product.staticTextExample.primaryText" defaultMessage="Lorem ipsum" />}
+                        secondaryText={<FormattedMessage id="product.staticTextExample.secondaryText" defaultMessage="Foo bar" />}
+                    />
+                );
+            },
+            flex: 1,
+            minWidth: 150,
+        },
+        {
+            field: "currencyAndNumber",
+            headerName: intl.formatMessage({ id: "product.currencyAndNumber", defaultMessage: "Price (currency and number)" }),
+            filterable: false,
+            sortable: false,
+            renderCell: ({ row }) => {
+                return (
+                    <GridCellContent
+                        primaryText={
+                            typeof row.price === "undefined" || row.price === null ? (
```

Comment:
You could also use `row.price != null`, which covers null and undefined

Reply:
More explicit also prevents someone from "refactoring" from `!=` to `!==` if they think that was a mistake 😄